test/net/sourceforge/kolmafia/objectpool/ConcoctionTest.java

```diff
@@ -1,11 +1,13 @@
 package net.sourceforge.kolmafia.objectpool;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.jupiter.api.Assertions.*;
```

Comment:
If it is reasonable to expand the includes that seem to be the standard.  Not blocking, mote along the lines of since you are already there.